### PR TITLE
Update orga slides for WT2425

### PR DESCRIPTION
**Pull request by uekerman**
## Description

- Updated both orga slides for WT 24/25.
- Added more details on grading.
- Tried to better underline that the contribution is a semester-long task (not only the last step).

@IshaanDesai @MakisH Please confirm that your are OK with everything or comment.

I will prepare the GitLab repo next (links on slides do not yet work).

Another feedback from last time was that students sometimes lost overview on where we were in the semester. I would suggest to show or integrate the current time table slide from `course_intro_slides.md` every week, e.g.

| Date | Type | Chapter | Topic | Lecturer |
| ---- | ---- | ------- |------ | -------- |
| 16.10. |Lecture | 0-1 | Course intro, intro to SSE, VC basics | Benjamin |
| 16.10. |Lecture | 1 | Git basics, my Git workflow, Git quiz, how to challenge  | Benjamin |
| **23.10.** |**Lecture** | **1** | **My neat little Git trick, merge vs rebase, working in teams**| **Benjamin** |
| 23.10. |Lab | 1 | Git  | Benjamin |
| 30.10. |Lecture | 2 | Intro containers, Docker, Singularity | Gerasimos |
| 30.10. |Lecture | 2 | Intro virtualization, VirtualBox, Vagrant | Gerasimos |
| 06.11. |Lab | 2 | Virtualization and containers | Gerasimos |
| 06.11. |Presentations | C | 1st student presentations | students |

### CODE REVIEW

**Review by IshaanDesai — APPROVED** [review]
Looks good :+1: some minor comments from my side.

File: 00_organization/challenge_intro_slides.md [hunk 1]
```diff
@@ -95,36 +113,36 @@ Rough workload / grading weights: 25%, 25%, 50%
 
 ## How to Submit my Choice?
 
-- [https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2223/challenge](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2223/challenge)
-- Comment in [issue #1](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2223/challenge/-/issues/1) till **27th of October** (next Thursday) evening (no FCFS)
+- [https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge)
+- Comment in [issue #1](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge/-/issues/1) till **23rd of October** (next Wednesday) evening (no FCFS)
 - Priority list with at least three choices
-- If not on our suggestion list, write short paragraph what the software does and give links
+- If **not** on our suggestion list, write short paragraph what the software does and give links
```
[the remark this answers is not in the document]
Comment (uekerman) [reply 00_organization/challenge_intro_slides.md:119]: In the same slides, 3 mins above :D

File: 00_organization/course_intro_slides.md [hunk 2]
```diff
@@ -196,37 +210,37 @@ td {
...
+    - Overall engagement (interactive lecture, discussions, small presentations, contributions, ...) (5%)
+- You will need to register yourself to the *"exam"* on C@MPUS.
+- Point of no return: Once you handed in the first report (Nov 6), you have to register.
 
 ---
 
 ## GitLab Account
 
-- Please write a mail till tonight to Ishaan
+- Please write a mail till tonight to Ishaan.
```
[the remark this answers is not in the document]
Comment (uekerman) [reply 00_organization/course_intro_slides.md:241]: My idea was that they should first write to you, but we will only create the accounts once we have the final list. Makes sense?

[the remark this answers is not in the document]
Comment (IshaanDesai) [reply 00_organization/course_intro_slides.md:241]: There would be a risk that someone who does not end up in the course writes to me and then I have to double check the final list before creating the accounts. This is not difficult, just added work. Can't we tell them about account creation when we have the final list?

**Review by MakisH — APPROVED** [review]
Dates and procedures look as expected :+1:
I will work on the CI in another PR.

File: 00_organization/course_intro_slides.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (uekerman) [reply 00_organization/course_intro_slides.md:241]: Mmh, I am afraid that we will anyway need to double check as students might drop out after writing to you.
(I could also do that on-the-fly as I will anyway need to sort out the applications)
Doing it like we currently describe gives us another check on how many series applications we have.

[the remark this answers is not in the document]
Comment (IshaanDesai) [reply 00_organization/course_intro_slides.md:241]: Okay

**Pull request merged**